CMakeLists.txt

```diff
@@ -722,28 +724,28 @@ SET(targets)
 
 IF(PCRE2_BUILD_PCRE2_8)
   IF(BUILD_STATIC_LIBS)
-    ADD_LIBRARY(pcre2-8-static STATIC ${PCRE2_HEADERS} ${PCRE2_SOURCES} ${PROJECT_BINARY_DIR}/config.h)
+    ADD_LIBRARY(pcre2-8-static STATIC ${PCRE2_SOURCES})
     SET_TARGET_PROPERTIES(pcre2-8-static PROPERTIES
       COMPILE_DEFINITIONS PCRE2_CODE_UNIT_WIDTH=8
+      PUBLIC_HEADER ${PCRE2_HEADERS}
       MACHO_COMPATIBILITY_VERSION "${LIBPCRE2_8_MACHO_COMPATIBILITY_VERSION}"
       MACHO_CURRENT_VERSION "${LIBPCRE2_8_MACHO_CURRENT_VERSION}"
       VERSION ${LIBPCRE2_8_VERSION}
       SOVERSION ${LIBPCRE2_8_SOVERSION})
     TARGET_COMPILE_DEFINITIONS(pcre2-8-static PUBLIC PCRE2_STATIC)
-    TARGET_INCLUDE_DIRECTORIES(pcre2-8-static PUBLIC ${PROJECT_BINARY_DIR})
```

Comment:
This line caused failures when exporting the targets and I had to remove it. I saw later that it was added in #113 to support `add_directory`ing the codebase from another project.

Is there a better way to satisfy both cases?

Reply:
Done, thanks!